impeller/renderer/backend/metal/render_pass_mtl.mm

```diff
@@ -440,6 +440,16 @@ static bool Bind(PassBindingsCache& pass,
           .zfar = v.depth_range.z_far,
       };
       [encoder setViewport:viewport];
+    } else {
+      MTLViewport viewport = {
```

Comment:
Minor nit: You can dry this up a bit. Instead of checking if the viewport has value, then getting its value and then converting to MTLViewport in two places, you could use `value_or` when getting the viewport. Then you won't need branch here and the code will be more concise.

Something like:

```
auto v = command.viewport.value_or({GetRenderTargetSize...});
// The old code remains as is
```

Reply:
Done